src/exporters/DRACOExporter.ts

```diff
@@ -12,50 +14,63 @@
  *  - exportNormals
  */
 
-/* global DracoEncoderModule */
```

Comment:
In the original version of `three.js`, undefined `DracoEncoderModules` were notified to Linter and others by writing `/* global DracoEncoderModules */`.

However, when I migrated `DRACOExporter` to TypeScript, I found that I could not use `/* global DracoEncoderModules */`, so I changed it to not check the type by adding `@ts-ignore`. Is this the correct way to handle this?